### ISSUE TITLE
Enabled Error

### ISSUE THREAD

**Issue by pixelinkmedia** [egtwobits/mesh_mesh_align_plus#29]
I installed the addon today,
Tried to enable the addon
Get this error... **_No module names mesh_mesh_algn_plus_**
I have Bv-3.6.2 on Win 10
![mapluserror1](https://github.com/egtwobits/mesh_mesh_align_plus/assets/51757955/182b1891-9f8a-4127-82c0-9f485704dd38)

Thanks

**Comment by ericsnekbytes** [comment]
Just tested on Blender 3.6.2 and MAPlus v1.0 on win11 and I am not seeing your error. Are you using the correct zip file? [Each post on the released page has zip files](https://github.com/egtwobits/mesh_mesh_align_plus/releases), near the end of the post.

**Comment by pixelinkmedia** [comment]
Yes, I used the "mesh_mesh_align_plus.zip" from Release page on github. It is v1.0 as pictured above

**Comment by ericsnekbytes** [comment]
I'm wondering why your path looks wrong in the error message. It says `addons\mesh_align_plus\__init__.py` when it should say `addons\mesh_mesh_align_plus\__init__.py` (the word mesh appears twice, which is the name of the package...which is so named because Blender requires the category to appear in front of the name).

**Comment by pixelinkmedia** [comment]
Well, I reinstalled it. And that worked.
However, changing a folder name (which is what I did) should not matter unless you are hardcoding the path in your script.
I have made 6 of my own addons and never have to worry about the folder name. I have changed not only my own addon folder names but, also other 3rd party ones because some devs put their version number on the folder. That's bad practice. So, when I see an unconventional folder name I rename it.

So, not sure what you are doing differently.
But, anyways it works. Thanks
